addons/calendar/models/calendar_event.py

```diff
@@ -824,6 +824,14 @@ def ics_datetime(idate, allday=False):
                 return idate.replace(tzinfo=pytz.timezone('UTC'))
             return False
 
+        def event_description(description):
```

Comment:
Because online events have fixed format for the description, this method removes some specific html tags, and converts it into readable plaintext (to be used in external calendars). Note that for regular (offline) events, we simply use the standard        `html2plaintext` method instead.

Reply:
But if salespeople update description of the event, won't that fail ?

Reply:
No it will not fail now as we are doing html2plaintext at return time. So if there are any other tags it will be replaced by it.

Reply:
So if I understand well: there is a "custom" html -> plaintext conversion and also a classic convert ? If that's the case, this is worth a comment somewhere to explain it :)

Reply:
We are converting custom html into normal html and then it into plaintext with help of classic convert. All this for data formatting in external calendar.
And also added doc string and in commit message.